### ISSUE TITLE
Install Janus config files from TinyPilot Debian package

### ISSUE THREAD

**Issue by mtlynch** [tiny-pilot/tinypilot#1496]
~~Blocked on https://github.com/tiny-pilot/tinypilot-pro/pull/985~~

We're currently placing Janus' config files using Ansible:

https://github.com/tiny-pilot/tinypilot/blob/d903d32f890505e97f94ec1c4549648037dc3555/ansible-role-ustreamer/tasks/install_janus.yml#L21-L32

It should be trivial to pull this logic out of Ansible and implement it instead in TinyPilot's Debian package.

Some notes on the migration:

* The Ansible task is a `template` task even though the source files don't have any templated values. I don't know if this was a mistake originally or if they used to have templates, but now they're just static files.
  * This makes the task easier, as we should be able to move the files like this:
    * `ansible-role-ustreamer/templates/janus.jcfg.j2` -> `debian-pkg/etc/janus/janus.jcfg`
    * `ansible-role-ustreamer/templates/janus.transport.websockets.jcfg.j2` -> `debian-pkg/etc/janus/janus.transport.websockets.jcfg`
* The Ansible task templatizes the path based on `ustreamer_janus_configs_dir`, but we can hardcode the path to `/etc/janus` for simplicity.
* We don't have to implement logic for restarting the Janus service.
* We should preserve `root` as the user and group.
* We should preserve `0644` as the permissions, but if that generates a CI error in lintian, we should accept lintian's suggestion. The important thing is that the execute bit is off for everyone and the write bit is off for non-owner.

## Similar work

* https://github.com/tiny-pilot/tinypilot/pull/1412

**Comment by jdeanwallace** [comment]
@mtlynch - Is this a duplicate of https://github.com/tiny-pilot/ustreamer-debian/issues/12?

Do we want to configure Janus in the TinyPilot or uStreamer Debian package?

**Comment by mtlynch** [comment]
>Do we want to configure Janus in the TinyPilot or uStreamer Debian package?

I'd like to do it in the TinyPilot Debian package because then it's all in a single repo. It's maybe more logical from the uStreamer Debian package, but then we're spreading more of our code across repos, which I think outweighs other concerns.

**Comment by mtlynch** [comment]
@db39 - Can you take this on next? This one is similar in process to https://github.com/tiny-pilot/tinypilot/issues/1467 but the process is simpler because we can do everything in a single change to the tinypilot repo rather than bouncing between this repo and the ustreamer-debian repo.

**Comment by mtlynch** [comment]
@db39 - I don't think we should have to explicitly set the permissions in `postinst`. I think `apt-get` just copies over the files with the right permissions, but double check after installing.
